core/src/main/scripts/importer/validateData.py

```diff
@@ -1346,6 +1348,30 @@ def checkSwissProt(self, value):
                 return True
         # if no reasons to return with a message were found, return valid
         return True
+    
+    def checkStartPosition(self, value):
+        """Check that the Start_Position value is an integer."""
+        if value.isdigit() == False or (value.isdigit() and '.' in value):
+            self.logger.error(
+                'The start position of this gene is not '
+                    'an integer',
+                extra={'line_number': self.line_number,
+                       'column_number': self.cols.index('Start_Position'),
+                       'cause': value})
+        # if no reasons to return with a message were found, return valid
+        return True
+    
+    def checkEndPosition(self, value):
+        """Check that the End_Position value is an integer."""
+        if value.isdigit() == False or (value.isdigit() and '.' in value):
+            self.logger.error(
+                'The end position of this gene is not '
```

Comment:
Same as in [my comment above](https://github.com/cBioPortal/cbioportal/pull/2649/files#r124505678): are you sure that this is about the end position of a gene?